src/components/presentational/DumbbellChart/DumbbellChart.tsx

```diff
@@ -0,0 +1,96 @@
+import "./DumbbellChart.css";
+import line from './horizontalLine.svg';
+import { useEffect, useState } from "react";
+import { debounce } from "lodash";
+import MyDataHelps from "@careevolution/mydatahelps-js";
+import Dumbbell from "./Dumbbell";
+import React from "react";
+
+export interface ClosedInterval { values : number[]};
+
+export interface DataPoint {
+    dataSet1 : ClosedInterval;
+    dataSet2 : ClosedInterval;
+}
+
+export interface Dumbbell {
+    dataPoint? : DataPoint;
+    xValue : string,
+    class? : DumbbellClass 
+}
+
+export interface Axis {
+    yRange : ClosedInterval;
+    yIncrement : number;
+    xIncrement : number;
+}
+
+export interface DumbBellChartProps {
+    axis : Axis;
+    dumbbells : Dumbbell[];
+}
+
+export enum DumbbellClass {
+    inRange,
+    outOfRange
+}
+
+export default function (props : DumbBellChartProps) {
+    const mdh = MyDataHelps;
+    const _minRange = props.axis.yRange.values[0];
+    const _maxRange = props.axis.yRange.values[1];
+    const _range = _maxRange - _minRange;
+
+    const [yAxisLines, setYAxisLines] = useState<any[]>([]);
+    const [chartDumbbells, setChartDumbbells] = useState<any[]>([]);
```

Comment:
This presentational component should need zero state...if you need a perf optimization (which you shouldn't) you could useMemo or something, but otherwise there's no reason to put stuff in state here